python/opae.admin/opae/admin/tools/pci_device.py

```diff
@@ -208,12 +208,18 @@ def add_subparser(subparser):
         unplug = subparser.add_parser('unplug')
         unplug.add_argument('-d', '--debug', action='store_true',
                             default=False, help='enable debug output')
-        #unplug.add_argument('-e', '--exclude', default=None,
-        #                    help='do not unplug the device specified here')
+        unplug.add_argument('-e', '--exclude', default=None, type=pci_devices,
+                            help='do not unplug the device specified here')
```

Comment:
Would we ever want to exclude more than one pci device?

Reply:
Hmm, I think we envisioned the context of this to be 1 workload PF and 1 PR PF. I can't imagine a situation where PR would require more than 1 device to exclude.  If you can think of one, then I can adjust the option to accept a list.

Reply:
If there were multiple PR regions (e.g. PF1 and PF2 independently PR'able), then we'd need multple --excludes. When that happens, we can extend --exclude for a list.